config/routes.rb

```diff
@@ -3,8 +3,6 @@
 require 'sidekiq_unique_jobs/web'
 require 'sidekiq-scheduler/web'
 
-Sidekiq::Web.set :session_secret, Rails.application.secrets[:secret_key_base]
```

Comment:
Also unrelated, but it was an annoying deprecation warning on boot!

Reply:
Mind putting that into another PR?